File: unit04-hw/multiplicationtable.rb
#! /usr/bin/ruby
# multiplicationtable.rb displays a table of logarithms
#
# Author: Robert Lowman
#
# Input:  The size of the multiplication table
# Precondition: The size must be greater than 0
# Output: A multiplication table of the given size


# Funtion that creates the multiplication table
# based on the given size
#
# Parameter: size The size of the multiplication table
# Precondition: size > 0
# Returns: void 
def DisplayTable(size)
  for i in 1..size
    for j in 1..size
      temp = i*j
      print "#{temp}\t"
    end
    puts " "
  end
end

# Where execution begins
if __FILE__ == $0
   print "Enter the size : "
  size = gets.chomp.to_i
  DisplayTable(size)
end

File: unit04-hw/multiplicationtable.py
## multiplicationtable.py displays a multiplication table.
##
## Input: size, the size of the table.
## PRE: start > 0.
## Output: A table of the given size
##
## Completed by: Robert Lowman
## Date: 12/1/2016

size = input('Enter the size: ')

for i in range(size):
    for j in range(size):
        value = (i + 1) * (j + 1)
        print str(value) + '\t',
    print ''
